src/main/java/com/example/librarymanagementsystem/utils/DatabaseConnection.java

```diff
@@ -0,0 +1,21 @@
+package com.example.librarymanagementsystem.utils;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DatabaseConnection {
+
+    private static final String URL = "jdbc:postgresql://localhost:5432/libraryManagementSys";
+    private static final String USERNAME = "postgres ";
```

Comment:
Don't pass sensitive information directly in your codebase. use environment variables to store the sensitive information and read the value of the environment variable in your codebase.